services/datamanager/datasync/sync.go

```diff
@@ -85,6 +88,10 @@ func (s *syncer) Close() {
 	_ = s.logger.Sync()
 }
 
+func (s *syncer) SetArbitraryFileTags(tags []string) {
```

Comment:
I added this instead of passing it into the Sync() function since that handles both data capture uploads and arbitrary uploads, and it felt odd to pass in an argument for only one. Open to thoughts otherwise.